dietpi/dietpi-software

```diff
@@ -5442,7 +5442,7 @@ _EOF_
 			fi
 
 			# Permissions: http://ompd.pl/configuration
-			G_EXEC chown -R www-data:www-data /var/www/ompd/{tmp,stream,cache}
+			G_EXEC chown -R www-data:www-data /var/www/ompd/{tmp,stream,cache,include}
```

Comment:
Adding `covers` according to: https://ompd.pl/configuration
```suggestion
			G_EXEC chown -R www-data:www-data /var/www/ompd/{tmp,stream,cache,covers,include/config.local.inc.php}
```
And as really only `config.local.inc.php` should be edited, hardening that further. Is there btw the plain text database password visible from the web interface? 🤔

Reply:
> Is there btw the plain text database password visible from the web interface? 🤔

yes unfortunately